linkerd/app/test/src/connect.rs

```diff
@@ -32,7 +32,7 @@ where
 
     fn call(&mut self, endpoint: E) -> Self::Future {
         let addr = endpoint.clone().into();
-        let span = tracing::info_span!("connect", %addr);
+        let span = tracing::debug_span!("connect", %addr);
```

Comment:
this is a test, and i think it's useful to have these traces in the test on by default